src/main/java/thePackmaster/actions/summonspack/JinxLoseHPAction.java

```diff
@@ -25,7 +25,7 @@ public JinxLoseHPAction(AbstractCreature target, AbstractCreature source, int am
 
     public void update() {
         if (duration == DURATION && target.currentHealth > 0 && !quiet)
-            CardCrawlGame.sound.play(SpireAnniversary5Mod.EVIL_KEY, 0.1f);
+            CardCrawlGame.sound.playAV(SpireAnniversary5Mod.EVIL_KEY, 0.1f, 0.5f);
```

Comment:
I made this quieter since the Calamity pack can apply it with a common, and I found it a bit too loud when it was going on regularly due to more debuffs being applied.